src/pipeline_migration/actions/format.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import logging
+from typing import Any
+
+from pipeline_migration.pipeline import PipelineFileOperation, iterate_files_or_dirs
+from pipeline_migration.types import FilePath
+from pipeline_migration.utils import YAMLStyle, BlockSequenceIndentation, dump_yaml
+
+logger = logging.getLogger("formatter")
+
+
+def register_cli(subparser) -> None:
+    format_parser: argparse.ArgumentParser = subparser.add_parser(
+        "format", help="Format given build pipelines."
+    )
+    format_parser.add_argument(
+        "file_or_dir",
+        nargs="*",
+        help="Specify locations from where finding out pipelines to add task. "
+        "A pipeline can be included in a PipelineRun or a single Pipeline definition. "
+        "%(prog)s searches pipelines from given locations by rules, if files are specified, "
+        "search just pipelines from them. If directories are specified, search YAML files from the "
```

Comment:
This description is for the `add-task` subcommand I think

Reply:
Yes, the search behavior is same for `format` command.

Reply:
Updated.